yacc/closure.c

```diff
@@ -110,8 +110,9 @@ void set_first_derives(void)
               mask = 1;
             }
         }
-
+#ifdef DEBUG
       vrow += varsetsize;
+#endif
```

Comment:
So it is - but in which case the line can be deleted (in the tidying up commit)...